### PR TITLE
refactor: simplify volume drawing and lower case variables

### PR DESCRIPTION
**Pull request by andiwand**

### CODE REVIEW

File: Core/src/Visualization/GeometryView3D.cpp [hunk 1]
```diff
@@ -191,42 +192,23 @@ void Acts::GeometryView3D::drawPortal(IVisualization3D& helper,
 
 void Acts::GeometryView3D::drawDetectorVolume(
     IVisualization3D& helper, const Experimental::DetectorVolume& volume,
-    const GeometryContext& gctx, bool drawSurfaces, const Transform3& transform,
```
Comment (benjaminhuth) [inline Core/src/Visualization/GeometryView3D.cpp:194]: So this boolean switch just has no use case and is therefore removed?

Comment (andiwand) [reply Core/src/Visualization/GeometryView3D.cpp:194]: yeah exactly. I think the surfaces should always be a part of the volume. whenever we want to have that optional we can add this back.

**Comment by andiwand** [comment]
@dimitra97 I adopted the code a little bit and removed the flag for now to simplify things

**Review by benjaminhuth — APPROVED** [review]
looks good to me

**Comment by dimitra97** [comment]
> @dimitra97 I adopted the code a little bit and removed the flag for now to simplify things

Looks good! More simplified indeed.

**Review by dimitra97 — APPROVED** [review]

**Pull request merged**